Sasquatch/Sasquatch/Info.plist

```diff
@@ -16,14 +16,12 @@
 	<string>APPL</string>
```

Comment:
@clpolet @jaelim-ms  So i'am deleting Info.plist in Sasquatch/ living two different plist files for Swift and ObjC versions?